projects/harfbuzz/hb-draw-fuzzer.options

```diff
@@ -0,0 +1,2 @@
+[libfuzzer]
```

Comment:
You want any time this fuzzer takes longer than 5 seconds to process an input to be filed as a bug?

Reply:
It is what our other fuzzers have also but sure we don't want infinite loop with the API, actually, I find one infinite loop which was a regression while developing this fuzzer and fixed it.

Reply:
ACK

Reply:
Thank you very much :)